frontend/src/main/components/Commons/Profits.js

```diff
@@ -1,24 +1,24 @@
 import React from "react";
 import { Card } from "react-bootstrap";
 import ProfitsTable from "main/components/Commons/ProfitsTable"
+import moment from "moment";
 
-const dummyData = [
-    { id: 1, profit: 10, date: "2021-03-05" },
-    { id: 2, profit: 11, date: "2021-03-06" },
-    { id: 3, profit: 10, date: "2021-03-07" },
-    { id: 4, profit: 8, date: "2021-03-08" }
-];
+function timestampToDate(timestamp) {
+    return moment(timestamp).format('YYYY-MM-DD');
```

Comment:
If this is the only thing we need moment for, let's see if we can find a native function that will do the trick:

https://stackoverflow.com/questions/3552461/how-to-format-a-javascript-date